### PR TITLE
migrate from CODEOWNERS to OWNERS

### PR DESCRIPTION
**Pull request by dennis-ge**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    - hanngos
+    - tillknuesting
+  service-management-approvers:
+    - wozniakjan
+    - piotrmiskiewicz
+    - szwedm
+    - ralikio
+    - hanngos
+    - tillknuesting
+
```
Comment (klaudiagrz) [inline OWNERS_ALIASES:59]:
```suggestion
    - PK85
```

**Review by klaudiagrz — COMMENTED** [review]
Adding PK to the approvers :D

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    - tillknuesting
+    - PK85
+  observability-reviewers:
+    - a-thaler
+    - rakesh-garimella
+    - skhalash
+    - chrkl
+    - lindnerby
+    - dennis-ge
+  observability-aprovers:
```
Comment (lindnerby) [inline OWNERS_ALIASES:67]: does this typo matter?

File: jaeger-operator/OWNERS [hunk 3]
```diff
@@ -0,0 +1,4 @@
+reviewers:
+  - observability-reviewers
+approvers:
+  - observability-approvers
```
Comment (lindnerby) [inline jaeger-operator/OWNERS:4]: i guess it does ;)

**Review by lindnerby — CHANGES_REQUESTED** [review]

File: jaeger-operator/OWNERS (hunk 3 above)
Comment (dennis-ge) [reply jaeger-operator/OWNERS:4]: yes it does :D thanks!

**Review by lindnerby — APPROVED** [review]

**Review by dbadura — APPROVED** [review]

**Review by wozniakjan — APPROVED** [review]

**Pull request merged**